# File: waywiser_perception/msg/BoundingBox.msg

geometry_msgs/Pose geometric_center_pose
float64 width
float64 height
float64 depth
